challenges/2021_CyrilMorcrette_cloudBaseHeight/cbh_challenge_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,2569 @@
+{
```

Comment:
<p>these comments could probably be in markdown. It might be nice to give a more detailed descripton (maybe in bullet points like at the top of the notebook, to give a text description of the code below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>